src/ui/components/atoms/label-variable/LabelVariable.test.jsx

```diff
@@ -0,0 +1,22 @@
+import { render } from '@testing-library/react';
+import LabelVariable from './LabelVariable.tsx';
+
+describe('LabelVariable Component', () => {
+  test('renders correctly with desktop props', () => {
+    const { getByText } = render(
+      <LabelVariable textbar1="." text1="RECURSOS" textbar2="." text2="PAGINA" />,
```

Comment:
This component needs to be a high-order component because should be the content of another component inside.
![image](https://github.com/Ditmar/OpenScience/assets/317151/0a0bb350-f90c-48fc-a53a-d0434eb4fd2b)
with this approach is not possible to do that